CMSgov/design-system
# [WNMGDS-781] Rename dense table

### Changed
- Rename `dense` to `compact` to match USWDS table

## How to test
- Check the Table component example to ensure that `dense` table is now referenced as `compact` table
- http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-781/rename-dense-table
